src/App.jsx

```diff
@@ -1,9 +1,92 @@
 import './App.css'
+import Button from "./components/button/Button.jsx";
+import Product from "./components/product/Product.jsx";
+import Tile from "./components/tile/Tile.jsx";
+import bag1 from "./assets/bag_1.png";
+import bag2 from "./assets/bag_2.png";
+import bag3 from "./assets/bag_3.png";
+import bag4 from "./assets/bag_4.png";
+import ourStoryImg from "./assets/our_story.png";
+import brandImg from "./assets/brand.png";
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  )
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button
+                    type='button'
```

Comment:
type zou je in dit geval ook in component kunnen zetten, omdat hij bij alle drie zelfde is. Maar als je het vaker wil gebruiker met andere types is dit inderdaad beter :)